Tests/test_GenBank.py

```diff
@@ -6663,6 +6663,22 @@ def test_multiline_structured_comment_parsing(self):
             "coastal water [ENVO:00002150]",
         )
 
+    def test_malformed_structured_comment_parsing(self):
+        """The comment will be ignored if it is not read by the parser
+           AYW00820.1; Malformed key-value delimiter used. Should be " :: ", but the record uses ": "
+        """
```

Comment:
We follow PEP257, which is what most of the flake8 messages are about:

```
flake8
./Tests/test_GenBank.py:6667:1: D205 1 blank line required between summary line and description
./Tests/test_GenBank.py:6667:1: D208 Docstring is over-indented
./Tests/test_GenBank.py:6667:1: D400 First line should end with a period
```

How about:

```python
    def test_malformed_structured_comment_parsing(self):
        """Test malformed structured comment gives warning.

        The comment will be ignored if it is not read by the parser AYW00820.1;
        Malformed key-value delimiter used. Should be " :: ", but the record uses ": "
        """
```